doc/api/buffer.markdown

```diff
@@ -127,7 +127,7 @@ Example:
 ### Class Method: Buffer.concat(list[, totalLength])
 
 * `list` {Array} List of Buffer objects to concat
-* `totalLength` {Number} Total length of the buffers when concatenated
+* `totalLength` {Number} Total length of the buffers list when concatenated
```

Comment:
But it's not the length of the list, is it? It's the total length of all buffers combined if I'm not mistaken.